ui/app/templates/components/token-expire-warning.hbs

```diff
@@ -1,28 +1,10 @@
-{{#unless (and isDismissed (is-before (now interval=1000) auth.tokenExpirationDate))}}
-  {{#if (is-after (now interval=1000) auth.tokenExpirationDate)}}
-    {{#message-in-page type="danger"}}
-      <div class="content">
-        <p>
-        Your auth token expired on {{moment-format auth.tokenExpirationDate 'MMMM Do YYYY, h:mm:ss a'}}. You will need to re-authenticate.
-        </p>
-      </div>
-      <button type="button" class="button" {{action "reauthenticate"}}>
-        Reauthenticate
-      </button>
-    {{/message-in-page}}
-  {{else if auth.allowExpiration}}
-    {{#message-in-page type="warning"}}
-      <div class="content">
-        <p>
-          We've stopped auto-renewing your current auth token due to inactivity.
-          Your token will expire in {{moment-from-now auth.tokenExpirationDate interval=1000 hideSuffix=true}} on
-          {{moment-format auth.tokenExpirationDate 'MMMM Do YYYY, h:mm:ss a'}}
-        </p>
-        <button type="button" class="button" {{action "renewToken"}}>Resume auto-renewal</button>
-      </div>
-      <button type="button" class="button is-transparent close-button" {{action "dismiss"}}>
-        {{i-con size=12 glyph="close"}}
-      </button>
-    {{/message-in-page}}
-  {{/if}}
-{{/unless}}
+<MessageInPage @type="danger">
+  <div class="content">
+    <p>
+      Your auth token expired on {{moment-format auth.tokenExpirationDate 'MMMM Do YYYY, h:mm:ss a'}}. You will need to re-authenticate.
+    </p>
+  </div>
+  {{#link-to "vault.cluster.logout" class="button"}}
+    Reauthenticate
+  {{/link-to}}
+</MessageInPage>
```

Comment:
I had moved `MessageInPage` to `AlertBanner`, so I'd think this is throwing an error. I left this file alone since the auto-renew message was moving, but we'll want to move this one.

Since I can't see how to do multiline suggestions:

```
<AlertBanner
  @type="danger"
  @message="Your auth token expired on {{moment-format auth.tokenExpirationDate 'MMMM Do YYYY, h:mm:ss a'}}. You will need to re-authenticate."
>
  {{#link-to "vault.cluster.logout" class="button link"}}
    Reauthenticate
  {{/link-to}}
</AlertBanner>
```

Reply:
Oh no 🙈- I caught the other one bug forgot to search for the angle bracket MessageInPage haha. Thanks for catching that!